### ISSUE TITLE
[Feature Request] Add decorators and store access to ErrorHandlers

### ISSUE THREAD

**Issue by brunoeduardodev** [elysiajs/elysia#134]
Currently, when adding a local or global error handler, you don't have access to store/decorators.

Having it would be pretty handy to make integrations with custom loggers, for example.

```ts
const app = new Elysia()
    .get('/error', () => {
        throw new Error("Ooops!")
    }, { 
      error: (handler) => {
        // I want to log something here :/
      }
    })
```

**Comment by brunoeduardodev** [comment]
will be fixed on 0.7

https://x.com/saltyAom/status/1701910215833124937

**Comment by SaltyAom** [comment]
Fixed in 0.7
